org.vranki.spectral.yaml

```diff
@@ -0,0 +1,26 @@
+id: org.vranki.spectral
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: spectral
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
```

Comment:
The app might need a cleanup section here to clean unneeded files

Reply:
Could you be more specific? Which files need to be cleaned? The .yaml is almost identical to the upstream Spectral yaml and it doesn't have a cleanup section.

Reply:
Here's a link to the documentation: https://docs.flatpak.org/en/latest/manifests.html?highlight=cleanup#cleanup

Reply:
I understand how the cleanup option works. Are there some files that need to be cleaned up after build?

Reply:
You can build Spectral IDF locally using flatpak-builder. You will see the build results in that directory so you can have an idea of which files can be removed. Might be some header files, unneeded icons or man pages (they are not exported)

Reply:
This is an Qt app, if the installation installs some files not needed it should be fixed in the .pro file. 

This is what spectral IDF currently installs:
```
├── current -> x86_64/test
└── x86_64
    └── test
        ├── 26558ad0f7aa1eec5fe6c3693e2efa7235dd405d24689bba7dae7d6052a93543
        │   ├── deploy
        │   ├── export
        │   │   ├── bin
        │   │   │   └── org.vranki.spectral
        │   │   └── share
        │   │       └── applications
        │   │           └── org.vranki.spectral.desktop
        │   ├── files
        │   │   ├── bin
        │   │   │   └── spectral
        │   │   ├── manifest.json
        │   │   └── share
        │   │       ├── appdata
        │   │       │   └── org.vranki.spectral.appdata.xml
        │   │       ├── app-info
        │   │       │   ├── icons
        │   │       │   │   └── flatpak
        │   │       │   │       ├── 128x128
        │   │       │   │       │   └── org.vranki.spectral.png
        │   │       │   │       └── 64x64
        │   │       │   │           └── org.vranki.spectral.png
        │   │       │   └── xmls
        │   │       │       └── org.vranki.spectral.xml.gz
        │   │       ├── applications
        │   │       │   └── org.vranki.spectral.desktop
        │   │       ├── icons
        │   │       │   └── hicolor
        │   │       │       ├── 128x128
        │   │       │       │   └── apps
        │   │       │       │       └── org.eu.encom.spectral.png
        │   │       │       ├── 16x16
        │   │       │       │   └── apps
        │   │       │       │       └── org.eu.encom.spectral.png
        │   │       │       ├── 256x256
        │   │       │       │   └── apps
        │   │       │       │       └── org.eu.encom.spectral.png
        │   │       │       ├── 32x32
        │   │       │       │   └── apps
        │   │       │       │       └── org.eu.encom.spectral.png
        │   │       │       └── 512x512
        │   │       │           └── apps
        │   │       │               └── org.eu.encom.spectral.png
        │   │       └── metainfo
        │   └── metadata
        └── active -> 26558ad0f7aa1eec5fe6c3693e2efa7235dd405d24689bba7dae7d6052a93543

33 directories, 16 files
```
I still don't understand which files need to be cleaned up. I don't see any unnecessary build artifacts. Anyway unnecessary installed files are a build issue, not a packaging issue.